etna/packages/etna-js/components/Nav.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react';
+require('./Nav.css');
+
+import Icon from './icon';
+
+const ICONS = {
+  superuser: 'user-ninja',
+  administrator: 'user-astronaut',
+  editor: 'user-edit',
+  viewer: 'user'
+};
+
+const Login = ({user}) => {
+  if (!user) return null;
+
+  let {first, last, permissions} = user;
+
+  let role = (permissions[CONFIG.project_name] || {}).role;
+
+  if (permissions.administration && permissions.administration.role == 'administrator') role = 'superuser';
+
+  return (
+    <div className='etna-login'>
+      {first} {last}
+      <Icon className='etna-user' icon={ ICONS[role] } title={role}/>
+    </div>
+  );
+};
+
+const Logo = ({LogoImage}) =>
+  <div className='etna-logo'>
+    <a href='/'>
+      <LogoImage/>
+    </a>
+  </div>;
+
+const Link = ({app}) => {
+  let image = <img title={app} className='etna-link' src={ `/images/${app}.svg` }/>;
+
+  let host_key = `${app}_host`;
+
+  if (!CONFIG[host_key]) return image;
+
+  let link = new URL(...[CONFIG.project_name, CONFIG[host_key]].filter(_=>_));
+
+  return <a href={link}>{image}</a>;
+}
+
+const Links = ({currentApp}) => {
+  let [ shown, setShown ] = useState(false);
+  let apps = [ 'timur', 'metis', 'janus' ].filter(a => a != currentApp);
+
+  if (!shown) return <div className='etna-links'>
+    <div className='etna-links-show' onClick={ () => setShown(true) } >
+      <Icon icon='chevron-circle-left'/>
+    </div>
+  </div>;
+
+  return <div className='etna-links'>
+    { apps.map( app => <Link key={app} app={app}/>) }
+    <div className='etna-links-hide' onClick={ () => setShown(false) } >
```

Comment:
Without your demo today, it's kind of inscrutable what this new arrow icon symbolizes in the Nav...the UX could be more intuitive. Would it help to include some hover text like `Other Apps` to indicate that it is an actionable button, or that it is hiding some information? Or, an alternative might be a more standardized menu icon, like the hamburger or waffle icon.

Reply:
I tried a hamburger there at first (there are no good waffle icons), but I wasn't sure it meant the right thing (the hamburger is usually a config menu). It also didn't quite have the feel of a drawer since the burger is so static. I put it back to burgerform, lemme know what you think.

Reply:
You're right, the hamburger looks funky because the menu slides out to the left, not down like it usually does. If it slid down instead, it would function more like the StackOverflow / StackExchange dropdown, where they show sister sites in a dropdown, or like the Office365 Waffle that drops down to show apps. Perhaps clicking your username drops down to reveal the other app icons?

I was thinking though, it's probably not a big deal to keep the arrow, too, since we have a small user base and it would be fairly simple to explain or point out, too.

Reply:
I tried another iteration, moving the burger to the left where a menu would be. I isolated the menu icon so it is less buttony in appearance to avoid confusion. The burger now opens a drawer downwards, which is not great since it covers up some important UI elements. However, it does seem like we'll want to have some settings stuff in a menu here at some point, so I think it seems worth putting in a drawer here.

Reply:
That looks nice! It does cover up some important elements, especially on Metis. If we get feedback that it is an issue (or especially if we add things to the menu and it starts covering up more items), we can always make other UI tweaks.